### PR TITLE
Gives lone infiltrator his micro-bomb back.

### PR DESCRIPTION
**Pull request by Adrian16199**
## About The Pull Request
Lone infiltrator gets their promised micro-bomb, yey.
## Why It's Good For The Game
They used to have it, was removed by skyrat. Either this or the paper should be removed.
My reasoning for why bring the micro-bomb instead of the paper being evaporated.
Most of the time, your actions get loud or you are taken as enemy of the corporation, causin you to be killed on sight or perhaps even apprehended into perma which will take only 5 minutes before getting executed due to the EOTC being applied.
Even if you would want to roleplay after being kidnapped, the chance is only there if they will actualy try to take you non-lethally so you shouldnt die a horrible death. 
You might say, why this then if you are going to die anyways? Well atleast this wont make you a lootbag just like the paper says and explains. You are there to infiltrate, if you fail then you are not worth loosing all that equipment to the enemy corporation, ey?
There is also the factor that some people dont want to sit thro 30 minutes of being dragged around to just be putted into empty perma and never be interacted ever again.
## Proof Of Testing
image
image
image
<details>
<summary>Screenshots/Videos</summary>

</details>

## Changelog
:cl:
balance: Lone infiltrator has been given back their micro-bomb as the syndicate intended. Budget cuts happend but we good now.
/:cl:

### CODE REVIEW

**Comment by nevimer** [comment]
eh, they're not really a nuclear operative...

**Comment by Adrian16199** [comment]
Why does everyone _say_ that

**Review by Swiftfeather — APPROVED** [review]
I agree wholeheartedly with this and generally refuse to play lone op because it doesn't have this.

**Comment by Adrian16199** [comment]
> Can you come up with something a bit more original?

...
okay...
How about we make it optional.
You choose if you want it implanted or not and an implanter just spawns in your modsuit.
Does that sound more *Original*

**Comment by Doom-external** [comment]
Could make it the tactical deniability implant so it doesn't damage the station.

**Comment by StrangeWeirdKitten** [comment]
I'm just going to make a vote for the maintainers to decide.

**Comment by Adrian16199** [comment]
Thank you

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]
Just spawn the implanter itself in their bag or on their person then I'll merge it.

**Comment by Adrian16199** [comment]
image
Done

File: modular_skyrat/modules/Midroundtraitor/code/event.dm [hunk 1]
```diff
@@ -70,7 +70,8 @@
 	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
 		/obj/item/tank/jetpack/oxygen/harness=1,\
 		/obj/item/gun/ballistic/automatic/pistol=1,\
-		/obj/item/knife/combat/survival)
+		/obj/item/knife/combat/survival,
```
Comment (StrangeWeirdKitten) [inline modular_skyrat/modules/Midroundtraitor/code/event.dm:73]:
```suggestion
		/obj/item/knife/combat/survival=1,\
```

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]

**Pull request merged**